src/main/java/com/zenjava/javafx/maven/plugin/AbstractJfxToolsMojo.java

```diff
@@ -73,7 +73,7 @@
      * <p>
      * This defaults to 'target/jfx/app' and in most cases there is no real need to change this.
      *
-     * @parameter property="jfxAppOutputDir" default-value="${project.build.directory}/jfx/app"
+     * @parameter property="jfx.jfxAppOutputDir" default-value="${project.build.directory}/jfx/app"
```

Comment:
;) you misunderstood me, i am using the invoker plugin to TEST my plugin ;) not for calling different plugins. The projects below `src/it/` are my test-projects, which are usable as example-projects aswell